ttml.go

```diff
@@ -33,6 +34,9 @@ var ttmlLanguageMapping = astimap.NewMap(ttmlLanguageEnglish, LanguageEnglish).
 // TTML regexp
 var ttmlRegexpDurationFrames = regexp.MustCompile("\\:[\\d]+$")
 
+// TTML offset time
+var ttmlRegexpOffsetTime = regexp.MustCompile("(\\d+)(\\.(\\d+))?(h|m|s|ms|f)$")
```

Comment:
Doc is mentionning `t` as well as `h|m|s|ms|f`, could you add it?

Reply:
Also I've the feeling the regexp should start with `^`, do you agree?

Reply:
Yes, start of string marker makes sense.

concerning ticks, i'm not quite sure how to implement it and if i'm not mistaken it would require for astisub to support a "tickrate" as well...... it would be a bit out of scope for this pull request

IMO a good compromise would be to throw a not implemented error when there is an offset-time in ticks

i'll update the PR soon :)